tck/src/main/java/org/eclipse/microprofile/openapi/tck/AirlinesAppTest.java

```diff
@@ -549,6 +549,8 @@ public void testSecurityRequirement(String type) {
                 allOf(aMapWithSize(2),
                         hasEntry(equalTo("userApiKey"), empty()),
                         hasEntry(equalTo("userBearerHttp"), empty()))));
+
+        vr.body("paths.'/zepplins/{id}'.delete.security.mutualTLSScheme[0][0]", equalTo("zepplinScope"));
```

Comment:
```suggestion
        vr.body("paths.'/zepplins/{id}'.delete.security[0].mutualTLSScheme[0]", equalTo("zepplinScope"));
``

Took me a moment to work out what had happened here. The `security` property contains an array, and so does the `mutualTLSScheme` property. However, since restassured aggregates all properties if you don't specify a list index, you ended up with a list of a list at the end.

I think it's clearer to move the list indices to their actual locations.

Reply:
I agree, just noting I got this idiom from many places in the same file so at some point it may be worth cleaning it up